src/D2L.Bmx/Program.cs

```diff
@@ -18,7 +18,7 @@
 	name: "--user",
 	description: ParameterDescriptions.User );
 
-// allow no-sandbox argument for chromium to for passwordless auth with elevated permissions
+// allow no-sandbox argument for Chromium to for passwordless auth with elevated permissions
```

Comment:
This message seems a bit weirdly written? 

Probably should say something along the lines of how: we need to set explicitly acknowledge we have no sandboxing in elevated permissions by passing this flag

Reply:
ha, I was actually trying to fix this comment, that's why I touched "Chromium" here, but didn't end up completing the job.
will leave it be for now.